docs/modules/plugins/partials/ui-size-and-coordinates-steps.adoc

```diff
@@ -0,0 +1,45 @@
+=== Save size and coordinates of element
+
+Saves the information about the size of an element and its coordinates relative to the viewport.
+
+The following table lists the size and coordinates related properties that can be accessed on a variable name using dot notation (please see examples section):
+
+[cols="1,3", options="header"]
+
+|===
+
+|Property
+|Description
+
+|`x`
+|the `x` coordinate
+
+|`y`
+|the `y` coordinate
+
+|`height`
+|the `height` of the element
+
+|`width`
+|the `width` of the element
+
+|===
+
+[source,gherkin]
+----
+When I save coordinates and size of element located by `$locator` to $scopes variable `$variableName`
+----
+
+* `$locator` - <<_locator>>.
+* `$scopes` - xref:commons:variables.adoc#_scopes[The comma-separated set of the variables scopes].
+* `$variableName` - The name of the variable to save the coordinates and size of an element.
+
+.Verify image size and coordinates
+[source,gherkin]
+----
+When I save coordinates and size of element located by `tagName(img)` to scenario variable `rect`
++Then `${rect.height}` is = `400`
++Then `${rect.width}` is = `400`
++Then `${rect.x}` is = `200`
++Then `${rect.y}` is = `8`
```

Comment:
let's use full notation in examples:
```
Then `${rect.height}` is equal to `400`
```
shortcuts are not always understood correctly